#Contains nodes planned path and time to reach those nodes
#Contains array of planned path node and time to reach each node

string agent_id

int32[] Pl_path_nodes

float64[] time_to_nodes
